includes/KSES.php

```diff
@@ -289,6 +290,8 @@ public function safecss_filter_attr( $css ) {
 				'overflow',
 				'vertical-align',
 				'list-style-type',
+
+				'z-index',
```

Comment:
```suggestion
				'z-index',
```

Reply:
All the entries in this array are grouped. The extra newline is on purpose.